spec.go

```diff
@@ -223,6 +229,9 @@ func createLibcontainerMount(cwd string, m specs.Mount) *configs.Mount {
 			source = filepath.Join(cwd, m.Source)
 		}
 	}
+	if m.Type == "cgroup" && flags&syscall.MS_RDONLY == 0 {
```

Comment:
OK, let Docker concern about this, I can live with that :) 
updated and squished the commits, PTAL.